File: src/test/java/easy/arrays/BestTimeToBuyAndSellStockTest.java
package easy.arrays;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

class BestTimeToBuyAndSellStockTest {

    @Test
    void maxProfit() {
        Assertions.assertEquals(5, new BestTimeToBuyAndSellStock().maxProfit(new int[]{7,1,5,3,6,4}));
    }
}

File: src/test/kotlin/arrays/BestTimeToBuyAndSellStockTest.kt
package arrays

import leetcode.easy.arrays.BestTimeToBuyAndSellStock
import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

class BestTimeToBuyAndSellStockTest {
    @Test
    fun `test for maximum profit value`(){
        assertEquals(5, BestTimeToBuyAndSellStock().maxProfit(intArrayOf(7,1,5,3,6,4)))
    }
}